tests/integration/test_storage_delta/test.py

```diff
@@ -471,7 +590,16 @@ def test_restart_broken(started_cluster, use_delta_kernel):
 
     write_delta_from_file(spark, parquet_data_path, f"/{TABLE_NAME}")
     upload_directory(minio_client, bucket, f"/{TABLE_NAME}", "")
-    create_delta_table(instance, TABLE_NAME, bucket=bucket, use_delta_kernel=use_delta_kernel)
+    # instance.query(get_creation_expression("s3", TABLE_NAME, started_cluster, use_delta_kernel = use_delta_kernel))
+
+    instance.query(
```

Comment:
Why do this change?

Reply:
In this one case, bucket name is different and I want to keep the create function generic for all storage types. 
We can make a generic function by passing url, but just seemed like too much modularity for single use (this is the only place we use it), so just wrote the statements, maybe we can add bucket/url option if we use it in other places too. please let me know your thoughts.